### ISSUE TITLE
ONRR impact expo - brainstorm topics 

### ISSUE THREAD

**Issue by Maroyafaied** [DOI-ONRR/onrr.gov-site#2592]
What do we want to provide employees during Impact Expo? 

Mission-related presentations about ongoing efforts happening in different parts of the organization.  
Engaging lunch-n-learns on personal growth and wellness.  
Opportunities to recognize colleagues/team accomplishments!
Recorded presentations for those who might miss any part of the week’s events.  
 

How can you contribute?  

We are looking for our very own ONRR team members to contribute to the success of the event, as well as speakers and topics to develop the content for the event as listed above. Please contact ONRR internal communication team by April 20, 2023, if you would like to discuss an idea for a topic or presentation

**Comment by cthomasONRR** [comment]
@Maroyafaied What do you think about a plain language presentation? I can go over the free tools l use and show before-and-afters from onrr.gov and nrrd. I'll be out of the office the entire week before the expo, but will be back the week of June 19. Also, June 19 is a federal holiday... I will message Christa to let her know. https://www.opm.gov/policy-data-oversight/pay-leave/federal-holidays/#url=2023

**Comment by Maroyafaied** [comment]
@cthomasONRR Yes, plain language is a good one. We did several presentations last year so we submit more than one topic. There's been interest in usability testing so I think that would be a good topic as well. We can all meet and brainstorm next sprint.

**Comment by lpgoldstein** [comment]
last year we presented on - agile, cx, NRRD overview, and the intern program - https://feedback.gsa.gov/jfe/form/SV_bejdlte7P5LpoBE

**Comment by eelzi-ONRR** [comment]
Ideas:

- User Research 101 - identifying problems, developing a research plan, user interview tips
- Accessibility Toolbox - we could go over the use and limitations of various tools that are available to everyone in ONRR (either freely online or part of the software we we all have)

**Comment by Maroyafaied** [comment]
I sent the plain language and user research 101 ideas.
